packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -448,6 +450,17 @@ MeetingUtil.startInternalStats = (meeting) => {
             // TODO: we don't need checks like this since it comes from the transceiver
             // refactor event linkage to use callbacks instead of events
             if (event.kind === 'audio' && event.stat === 'bytesSent') {
+              Trigger.trigger(
+                this,
```

Comment:
After pulling down and testing locally, I was not getting any of these events and lots of console parameter errors.

Diving in, I found that you need to use `meeting` here instead of `this`, since these methods are class static methods. 

```suggestion
                meeting,
```